src/ui/ui_nav_link.js

```diff
@@ -0,0 +1,16 @@
+var _ = require('lodash');
```

Comment:
Can you switch these to es6 imports?

Also, only import `pick` since that's all you use?